aggregation-worker/Dockerfile

```diff
@@ -18,7 +18,9 @@ RUN \
   rm -rf /var/lib/apt/lists/*
 
 # copy aggregator to image
-COPY bin/* /opt/aggregator/
```

Comment:
Would it be better for the docker stuff to be in src/main/docker (using the contextDirectory configuration parameter) for consistency with other project source code?

Reply:
To me the dockerfile make more sense at the top level next to pom.xml. 

I believe other people are doing the same. I was not able to find any reference to best practice guide.

https://github.com/spotify/dockerfile-maven/tree/master/plugin/src/it

Reply:
maven conventions put source files in the src directory - I wouldn't like to see the project directory getting polluted with source code - its not as much of an issue now that the aggregator.sh is gone.